client/src/utils/actions.ts

```diff
@@ -1,14 +1,26 @@
-const identity = func => func;
+// Returns the value of the first argument. All others are ignored.
+function identity<T extends any[]>(...func: T): T[0] {
```

Comment:
Note that this does exactly the same thing as the last implementation. Just written out like this to allow it to accept a splat and pass through the type of the first argument